src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -478,7 +478,10 @@ protected override SelectionState PopulateItemsCore(string currentSelection)
                 if (category != null)
                 {
                     string name = getFullName(category);
-                    Items.Add(new DynamoDropDownItem(name, categoryId));
+                    if (name != "Cameras")
```

Comment:
Looks like this is obsoleted already but I will make a couple of comments anyway. First, it is a bad idea to filter by name. This string is localized so this will not work on non-English versions of Revit. It has to be based on the category Id. Second, we are aware of that some of the categories we expose in this list doesn't make sense and are not exposed in the Revit UI today because they  are internal categories. The problem is that there is no way right now to know if a category is internal because those properties has not been exposed in the public revit api (yet). I don't think we want our own, mirrored,  by category filtering mechanism on the Dynamo side. We want Revit to provide that information and until then I think that it is OK as it is right now.